prover/crates/bin/prover_job_monitor/src/queue_reporter/proof_compressor_queue_reporter.rs

```diff
@@ -57,10 +57,11 @@ impl Task for ProofCompressorQueueReporter {
             .get_oldest_not_compressed_batch()
             .await;
 
-        if let Some(l1_batch_number) = oldest_not_compressed_batch {
+        //todo: this metric might be revisited
```

Comment:
grammar nit: `//todo: this metric may need to be revisited`.

Apart from that, let's revamp the metrics in the next iteration